sqlsynthgen/make.py

```diff
@@ -256,12 +256,15 @@ def make_src_stats(
     else:
         engine = create_engine(dsn, echo=False, future=True)
 
-    dp_config = config.get("smartnoise-sql", {})
-    snsql_metadata = {"": dp_config}
-    src_stats = {}
-    for stat_data in config.get("src-stats", []):
-        privacy = snsql.Privacy(epsilon=stat_data["epsilon"], delta=stat_data["delta"])
-        with engine.connect() as conn:
+    use_smartnoise_sql = config.get("use-smartnoise-sql", True)
```

Comment:
As discussed, this is a different string to the one in the tests:

```suggestion
    use_smartnoise_sql = config.get("use-smartnoise-sql", True)
```

Reply:
Fixed the one in tests.